uint8 PWM_OUTPUT_MAX_CHANNELS = 16
uint16[16] values
uint32 channel_count
